resources/lang/en/general.php

```diff
@@ -91,7 +91,6 @@
     'debug_warning'         => 'Warning!',
     'debug_warning_text'    => 'This application is running in production mode with debugging enabled. This can expose sensitive data if your application is accessible to the outside world. Disable debug mode by setting the <code>APP_DEBUG</code> value in your <code>.env</code> file to <code>false</code>.',
     'delete'  				=> 'Delete',
-    'delete_confirm'  		=> 'Are you sure you wish to delete :item?',
```

Comment:
Why is this deleted?

Reply:
Oh, I thought I added this back in??

I originally thought we didn't need it cause you said we had a string already, but this was probably the strong you were talking about.